client/content_publishing_report/controllers/ContentPublishingReportController.js

```diff
@@ -376,8 +373,21 @@ export const generateTitle = (chart, params) => {
     return gettext('Published Stories per {{group}}', {group: parentName});
 };
 
+export function getCustomVocabFieldName(field) {
+    const obj = JSON.parse(field);
+
+    return getCustomVocabulariesData().find(
+        (value) => value.qcode.scheme == obj.scheme)?.name;
```

Comment:
`)?.name;` should be on a new line, as the associated `(` is on a separate line (this makes it easier to read order and group of operations)